bulletin/index.html

```diff
@@ -114,16 +113,16 @@ <h4 class="buarticle-subheading">MACILE Spanish currilum</h4>
 			</article>
 			<article class="buarticle">
 				<img src="/images/bulletin/group-2008-014.jpg">
-				<p>MACILE <i>nourishes the innate curiosity and wonderful imagination of young people rather than discourage them</i>, as Einstein admonished, to expand their horizons.</p>
+				<p>MACILE strives to expand the young people's horizons,<i>nourishing their innate curiosity and wonderful imagination rather than discouraging them</i>, as Einstein admonished.</p>
 			</article>
 			<article class="buarticle">
 				<img src="/images/bulletin/busLeaders-2008-m.jpg">
-				<p>Visit from Dominican business leader and gov officials. They spoke about the job markets and the government scholarship program. Students learned about opportunities and requirements to be successful.</p>
+				<p>Visit from a Dominican business leader and a government official. They spoke about the job market and the government scholarship program. Students learned about opportunities and requirements for success.</p>
 			</article>
```

Comment:
Here you made the fix and then you reverted it in the next commit 🤦‍♀️

Reply:
"for success" .... in what...it was better before.

Reply:
Done.